ksz8863/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: 0.2.8
+version: 0.2.9
```

Comment:
Please do not update manually. It will be updated automatically during version bump.